### PR TITLE
fix test_route_map beaker test for I9

### PR DESCRIPTION
**Pull request by rahushen**
Fixes the test_route_map ordering issue on 9k.

### CODE REVIEW

File: tests/beaker_tests/cisco_route_map/test_route_map.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 # In I7 match_src_proto order is not maintained in running config.
 # This behavior is currently observed only on the N9K.
-if platform[/n9k/] && image_version.to_s.strip[/I7/]
+if platform[/n9k/] && image_version.to_s.strip[/I(7|8|9)/]
```
Comment (saichint) [inline tests/beaker_tests/cisco_route_map/test_route_map.rb]: Does this work for switches running 9.2.1 code? (One of my switch is running 9.2.1 and it fails there). There is no 'I9' in the show version for that switch.

Comment (rahushen) [reply tests/beaker_tests/cisco_route_map/test_route_map.rb]: Yes, works for me with this image on:
NXOS: version 9.2(1)

Comment (saichint) [reply tests/beaker_tests/cisco_route_map/test_route_map.rb]: Ok, In that case  👍

**Comment by saichint** [comment]
👍

**Pull request merged**